src/main/java/com/khutircraftubackend/auth/AuthResponseMessages.java

```diff
@@ -0,0 +1,18 @@
+package com.khutircraftubackend.auth;
+
+public final class AuthResponseMessages {
+
+    public static final String EMAIL_NOT_FOUND = "Користувач з таким email : %s не знайдений.";
+    public static final String EMAIL_DISABLED = "Підтвердження за поштою %s не пройдено.";
```

Comment:
similar to Auth Service.
it's better to replace "Disabled" here with "Blocked" or similar.